lib/inc/drogon/HttpAppFramework.h

```diff
@@ -578,6 +581,14 @@ class DROGON_EXPORT HttpAppFramework : public trantor::NonCopyable
         return *this;
     }
 
+    /// Register a default handler into the framework when no handler matches
+    /// the request. If set, it replaces the static file router.
+    /**
+     * @param function indicates any type of callable object with a valid
+     * processing interface.
+     */
+    virtual HttpAppFramework &setDefaultHandler(DefaultHandler handler) = 0;
+
```

Comment:
So the default handler blocks accessing to static files, is there any better way?

Reply:
How do you think it should behave ? 

I can propose to call the DefaultHandler at the end, if the StaticFileRouter does not find a file:

in ``

```c++
void StaticFileRouter::route(
    const HttpRequestImplPtr &req,
    std::function<void(const HttpResponsePtr &)> &&callback)
{
    [...]
    if (defaultHandler) {
      defaultHandler(req, std::move(callback));
    } else {
      callback(HttpResponse::newNotFoundResponse());
    }
}
```

Is it ok for you then ?

Reply:
I think this is better

Reply:
Ok then, I updated in c6997eb8